### PR TITLE
Make text-shadow and box-shadow use SimpleShadow

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/17543)

### CODE REVIEW

**Comment by nox** [comment]
r? @emilio 
@bors-servo try
https://treeherder.mozilla.org/#/jobs?repo=try&revision=238dff57cad00db63a1f0e50bc0d03fad13667fe

File: components/style/values/specified/effects.rs [hunk 1]
```diff
@@ -42,6 +47,85 @@ pub enum Factor {
 /// A specified value for the `drop-shadow()` filter.
 pub type SimpleShadow = GenericSimpleShadow<Option<Color>, Length, Option<Length>>;
 
+impl Parse for BoxShadow {
+    fn parse<'i, 't>(
+        context: &ParserContext,
+        input: &mut Parser<'i, 't>)
+    -> Result<Self, ParseError<'i>> {
```
Comment (emilio) [inline components/style/values/specified/effects.rs]: nit: Paren should be in this line.

**Review by emilio — APPROVED** [review]

**Comment by mantaroh** [comment]
Hi @nox,

This changes will cause difference of serialization from gecko.
i.e. The serialization result of gecko is `length length ... color` , but Stylo is `color length length ...`.

In gecko, we have the web-animation's keyframe test using serialized result, this change will cause this test fail.

Do we need to fix to the same result as gecko?

**Comment by nox** [comment]
I think we should fix Gecko, which has a different serialisation order for specified and computed values.

**Pull request merged**